packages/nds-forms/package.json

```
@@ -0,0 +1,35 @@
+{
+  "name": "@nice-digital/nds-forms",
+  "version": "1.0.0",
+  "description": "> TODO: description",
```

Please add a description e.g.:
```suggestion
  "description": "Form components for the NICE Design System",
```